pkgs/development/python-modules/pysqueezebox/default.nix

```diff
@@ -0,0 +1,20 @@
+{ stdenv, fetchPypi, buildPythonPackage }:
+
+buildPythonPackage rec {
+  pname = "pysqueezebox";
+  version = "0.4.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "02d73e98314a63a38c314d40942a0b098fb59d2f08ac39b2627cfa73f785cf0d";
+  };
+
+  doCheck = false;
```

Comment:
If tests are not available, then please use `pythonImportsCheck` to import the most important modules. This isn't as good as unit tests, but will usually give a good indication of run-time errors. Please see [pythonImportsCheck documentation](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pythonimportscheck) for more information.

and please comment as to why tests were disabled.

Reply:
The tests are not available in the Pypi tar.gz

Reply:
Have you tried pulling the package from GitHub then? Having the tests would be much better.

Reply:
[According to the dev](https://github.com/rajlaud/pysqueezebox/pull/2#issuecomment-684119583), the tests are a work in progress. That's why they are not included.

Reply:
Fair enough.